sdcm/logcollector.py

```diff
@@ -43,6 +43,9 @@
 from sdcm.utils.docker_utils import get_docker_bridge_gateway
 
 
+import cluster_cloud
```

Comment:
What do you think about moving this import to `get_aws_instances_by_testid()` method?